### PR TITLE
ASLR: Embedded data

### PR DESCRIPTION
**Pull request by jenswi-linaro**

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
There's some unexpected errors on PLATFORM=stm-b2260, I'm working on fixing them.

**Comment by jenswi-linaro** [comment]
Updated

**Comment by jenswi-linaro** [comment]
Update commit message in "core: move relocation to embedded data region" and applied tag.
Note that "core: add embedded data region" and "core: generic_entry_a64.S: fix boot_mmu_config alignment" doesn't have any tags yet.

File: core/arch/arm/kernel/generic_entry_a64.S [hunk 1]
```diff
@@ -338,6 +338,7 @@ LOCAL_FUNC enable_mmu , : , .identity_map
 	ret
 END_FUNC enable_mmu
 
+	.balign	8
```
Comment (jforissier) [inline core/arch/arm/kernel/generic_entry_a64.S]: Why is it important?

**Comment by jforissier** [comment]
For "core: add embedded data region": LGTM.

`Acked-by: Jerome Forissier <jerome@forissier.org>`

File: core/arch/arm/kernel/generic_entry_a64.S (hunk 1 above)
Comment (jenswi-linaro) [reply core/arch/arm/kernel/generic_entry_a64.S]: If alignment is wrong we could get an alignment error. I'm I bit surprised that I didn't catch it as an error before, now I just took by inspection (I had an alignment issue on a 8 bytes variable in another PR).

Comment (jforissier) [reply core/arch/arm/kernel/generic_entry_a64.S]: Ah, right, it is a data structure. For this commit:
`Reviewed-by: Jerome Forissier <jerome@forissier.org>`

**Comment by jenswi-linaro** [comment]
Tag applied

**Pull request merged**